pkg/process/monitor/process_monitor.go

```diff
@@ -259,6 +259,7 @@ func (pm *ProcessMonitor) Initialize() error {
 						pm.evalEXECCallback(c, ev.ProcessPid)
 					}
 				case *netlink.ExitProcEvent:
+					log.Debugf("Got exit event for pid: %d", ev.ProcessPid)
```

Comment:
let's move it / add another one for the callback itself

Reply:
Added